.pipelines/build.cmd

```diff
@@ -33,9 +33,32 @@ echo Copying contents...
 cd /D "%~dp0"
 if errorlevel 1 goto :error
 
-xcopy /e /i c:\webrtc\src\out ..\output
-if errorlevel 1 goto :error
+call :copy c:\webrtc\src\out\msvc\uwp\Release\x64\obj\webrtc.lib ..\output\msvc\uwp\Release\x64\obj\
+call :copy c:\webrtc\src\api\*.h ..\include\api\
+call :copy c:\webrtc\src\audio\*.h ..\include\audio\
+call :copy c:\webrtc\src\base\*.h ..\include\base\
+call :copy c:\webrtc\src\call\*.h ..\include\call\
+call :copy c:\webrtc\src\common_audio\*.h ..\include\common_audio\
+call :copy c:\webrtc\src\common_video\*.h ..\include\common_video\
+call :copy c:\webrtc\src\data\*.h ..\include\data\
+call :copy c:\webrtc\src\logging\*.h ..\include\logging\
+call :copy c:\webrtc\src\media\*.h ..\include\media\
+call :copy c:\webrtc\src\modules\*.h ..\include\modules\
+call :copy c:\webrtc\src\p2p\*.h ..\include\p2p\
+call :copy c:\webrtc\src\pc\*.h ..\include\pc\
+call :copy c:\webrtc\src\rtc_base\*.h ..\include\rtc_base\
+call :copy c:\webrtc\src\rtc_tools\*.h ..\include\rtc_tools\
+call :copy c:\webrtc\src\sdk\*.h ..\include\sdk\
+call :copy c:\webrtc\src\stats\*.h ..\include\stats\
+call :copy c:\webrtc\src\system_wrappers\*.h ..\include\system_wrappers\
+call :copy c:\webrtc\src\video\*.h ..\include\video\
+call :copy c:\webrtc\src\common_types.h ..\include\
+
+goto :exit
 
+:copy
```

Comment:
The name of this function is confusing. There is a cmd command named `copy`. How about renaming this function to `copyHeaders` ?

Reply:
Renamed to copyFiles. We are also copying .lib file.